tests/foreman/api/test_errata.py

```diff
@@ -737,71 +737,81 @@ def test_positive_get_diff_for_cv_envs(self):
-            2. Multiple environments/content views present.
-
-        :Steps: POST /katello/api/hosts/bulk/available_incremental_updates
-
-        :expectedresults: Selected errata are applied to multiple content views
-            in multiple environments.
-
-        :CaseAutomation: notautomated
-
-        :CaseLevel: System
-        """
-
-    @pytest.mark.stubbed
-    @tier3
-    def test_positive_incremental_update_query_envs_cvs(self):
-        """Query a subset of environments or content views to push new
-        errata
+            1. Errata synced on satellite server
 
-        :id: f6ec8066-36cc-42a8-9a1a-156721e733c3
+        :Steps:
+            1. Create VM as Content Host, registering to CV with custom errata
+            2. Install package in VM so it needs one erratum
+            3. Check if incremental_updates required:
+                POST /api/hosts/bulk/available_incremental_updates
+            4. Assert empty [] result (no incremental update required)
+            5. Apply a filter to the CV so errata will be applicable but not installable
+            6. Publish the new version
+            7. Promote the new version into the same LCE
+            8. Check if incremental_updates required:
+                POST /api/hosts/bulk/available_incremental_updates
+            9. Assert incremental update is suggested
 
-        :Setup:
-            1. Errata synced on satellite server.
-            2. Multiple environments/content views present.
-
-        :Steps: POST /katello/api/content_view_versions/incremental_update
 
-        :expectedresults: Subset of environments/content views retrieved.
-
-        :CaseAutomation: notautomated
-
-        :CaseLevel: System
-        """
-
-    @upgrade
-    @pytest.mark.stubbed
-    @tier3
-    def test_positive_incremental_update_apply_packages_to_envs_cvs(self):
-        """Select multiple packages and apply them to multiple content
-        views in multiple environments
-
-        :id: 61549360-ce99-42a3-8d6b-2cd713f8b556
-
-        :Setup:
-            1. Errata synced on satellite server.
-            2. Multiple environments/content views present.
-
-        :Steps: POST /katello/api/content_view_versions/incremental_update
-
-        :expectedresults: Packages are applied to multiple environments/content
-            views.
-
-        :CaseAutomation: notautomated
+        :expectedresults: Incremental update requirement is detected.
 
         :CaseLevel: System
         """
+        with VirtualMachine(distro=DISTRO_RHEL7) as client_vm:
+            client_vm.install_katello_ca()
+            client_vm.register_contenthost(self.org.label, self.activation_key.name)
+            self.assertTrue(client_vm.subscribed)
+            client_vm.enable_repo(REPOS['rhst7']['id'])
+            client_vm.install_katello_agent()
+            host = entities.Host().search(query={'search': 'name={0}'.format(client_vm.hostname)})[
+                0
+            ]
```

Comment:
If you used f-string instead of `format`, would that all fit in single line?

Reply:
No, it would be like so:

            host = entities.Host().search(query={'search': f'name={client_vm.hostname}'})[
                0
            ]

Reply:
I was wrong, black reformats it:
+            host = entities.Host().search(query={'search': f'name={client_vm.hostname}'})[0]

Reply:
change made, force pushed, new test running

Reply:
Great!